source/samples/pubsub/PubSubFeature.cpp

```diff
@@ -29,6 +32,10 @@ constexpr char PubSubFeature::DEFAULT_PUBLISH_FILE[];
 constexpr char PubSubFeature::DEFAULT_SUBSCRIBE_FILE[];
 
 #define MAX_IOT_CORE_MQTT_MESSAGE_SIZE_BYTES 128000
+#define MAX_EVENTS 1000                           /* Maximum number of events to process*/
+#define LEN_NAME 16                               /* Assuming that the length of the filename won't exceed 16 bytes*/
+#define EVENT_SIZE (sizeof(struct inotify_event)) /*size of one event*/
+#define EVENT_BUFSIZE (MAX_EVENTS * (EVENT_SIZE + LEN_NAME)) /*size of buffer used to store the data of events*/
```

Comment:
Suggest using constexpr instead of a macro.. Unless there is a good reason for one.